docs/en/ingest-management/elastic-agent/elastic-agent-unprivileged-mode.asciidoc

```diff
@@ -33,9 +33,25 @@ elastic-agent install \
   --unprivileged
 ----
 
-IMPORTANT: On Linux systems, once {agent} has been installed with the `--unprivileged` flag, all {agent} commands that you run should not be prefixed with `sudo`.
-Including `sudo` in a command may result in <<agent-sudo-error,an error>> due to the agent not having the required privileges.
-The `sudo` option is still required for the `elastic-agent install` command.
+[IMPORTANT] 
+====
+Note the following current restrictions for running {agent} in `unprivileged` mode:
+
+* On Linux systems, after {agent} has been installed with the `--unprivileged` flag, all {agent} commands can be run without being the root user.
+** The `sudo` option is still required for the `elastic-agent install` command.
+Only `root` can install new services.
+The installed service will not run as the root user.
+* Using `sudo` without specifying an alternate non-root user with `sudo -u` in a command may result in <<agent-sudo-error,an error>> due to the agent not having the required privileges.
+* Using `sudo -u elastic-agent-user` will run commands as the user running the {agent} service and will always work.
+* For files that allow users in the `elastic-agent` group access, using an alternate user that has been added to that group will also work.
+There are still some commands that are only accessible to the `elastic-agent-user` that runs the service.
+** For example, `elastic-agent inspect` requires you to prefix the command with `sudo -u elastic-agent-user`.
```

Comment:
imo, examples shouldn't be included as child bullets, but instead as paragraphs in the bullet

Reply:
Yeah, good point! I'll leave as is for now though.